xmtp_mls/src/groups/group_metadata.rs

```diff
@@ -67,6 +67,7 @@ impl GroupMetadata {
     pub(crate) fn to_proto(&self) -> Result<GroupMetadataProto, GroupMetadataError> {
         let conversation_type: ConversationTypeProto = self.conversation_type.clone().into();
         Ok(GroupMetadataProto {
+            creator_inbox_id: "".to_string(),
```

Comment:
temporary placeholder, as https://github.com/xmtp/proto/commit/1ea946c9ef79197b32e7bc88696e6423ecf3b7b6  introduced `creator_inbox_id`

Reply:
My apologies. [This PR](https://github.com/xmtp/libxmtp/pull/699) is just waiting for reviews

Reply:
Let's merge this one first and I'll deal with the conflicts.